### PR TITLE
feat: UI (#13)

### PR DESCRIPTION
**Pull request by xezno**
Basic user interface.

image

The following has been hooked up to the game backend:
- Player info / player list (aside from total worm health)
- Turn timer
- Game timer
- Worm nametags

The following is still to-do and is either waiting for the feature to actually be implemented or refactored:
- Total worm health for a player
- Inventory
- Wind direction

### CODE REVIEW

File: code/States/StateHandler.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public void ChangeState( BaseState state )
 		[Event.Tick.Server]
 		private void Tick()
 		{
+			Game.Instance.State = State;
```
Comment (Matt9440) [inline code/States/StateHandler.cs:38]: You shouldn't need to do this anymore as the game state networks properly.

**Review by Matt9440 — CHANGES_REQUESTED** [review]
There's a few conflicts that need resolving and State is being set in Tick which we don't need to do anymore. Other than those minors things, looks great! Well done

File: code/States/StateHandler.cs (hunk 1 above)
Comment (xezno) [reply code/States/StateHandler.cs:38]: fair

Comment (xezno) [reply code/States/StateHandler.cs]: Should be fixed now

File: code/States/StateHandler.cs [hunk 2]
```diff
@@ -31,12 +31,5 @@ public void ChangeState( BaseState state )
 			State = state;
 			State?.Start();
 		}
-
-		[Event.Tick.Server]
-		private void Tick()
-		{
-			Game.Instance.State = State;
-			State?.OnTick();
```
Comment (Matt9440) [inline code/States/StateHandler.cs:38]: I think we still need to be ticking the state, we just don't need to be setting it each tick. I'll merge this in and reimplement the OnTick

**Review by Matt9440 — APPROVED** [review]

**Pull request merged**